cmd/healthchecker/options/options.go

```diff
@@ -47,8 +48,12 @@ type HealthCheckerOptions struct {
 func (hco *HealthCheckerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&hco.Component, "component", types.KubeletComponent,
 		"The component to check health for. Supports kubelet, docker and cri")
-	fs.StringVar(&hco.SystemdService, "systemd-service", "",
-		"The underlying systemd service responsible for the component. Set to the corresponding component for docker and kubelet, containerd for cri.")
+	// Deprecated: For backward compatibility on linux environment. Going forward "service" will be used instead of systemd-service
+	if runtime.GOOS == "linux" {
+		fs.MarkDeprecated("systemd-service", "please use --service flag instead")
+	}
+	fs.StringVar(&hco.Service, "service", "",
+		"The underlying service responsible for the component. Set to the corresponding component for docker and kubelet, containerd for cri.")
```

Comment:
We should explain what the service means on linux and windows. And I don't think `cri` is a valid option here.

Reply:
What do you mean cri is not a valid option? I think the message says to use containerd for cri?

Reply:
Oh nvm. I misread the flag description.